Resources/Machine Learning/qml/mlClassificationLda.qml

```diff
@@ -213,10 +213,27 @@ Form {
     }
 
     Item {
-        height: 			saveModel.height
+        height: 			addClasses.height*2
         Layout.fillWidth: 	true
         Layout.columnSpan: 2
 
+        CheckBox {
```

Comment:
A general point on the braces, please put the opening brace on their own line in the future. Not a blocker or anything though.